lib/Drupal/service_container/Messenger/MessengerInterface.php

```diff
@@ -0,0 +1,81 @@
+<?php
```

Comment:
I think moving to src/Messenger is better.

Tests had only been in lib/Drupal/service_container, because simpletest in D7 was missing PSR-4 support. It has that now, so for newer versions of Drupal we can switch over.